tests/security/cc/ip_eb_tables.pm

```diff
@@ -24,6 +24,9 @@ sub run {
 
     select_console 'root-console';
 
+    # install command ping
+    zypper_call('in iputils');
```

Comment:
I'd suggest, you add this into cc_setup test module. iputils should be used by audit-remote test as well. I would consider it a "basic" cmd :)

Reply:
Done.